api/.env.example

```diff
@@ -17,7 +17,10 @@ AUTH_CODE_SECRET=<secret>
 SESSION_SECRET=<secret>
 DB_CONNECT=mongodb+srv://<DB_USERNAME>:<DB_PASSWORD>@<CLUSTER>/<DB_NAME>?retryWrites=true&w=majority
 
+RUN_TIMES=[sas|js|sas,js|js,sas] default considered as sas,js
 SAS_PATH=/opt/sas/sas9/SASHome/SASFoundation/9.4/sas
+NODE_PATH=/Users/username/.nvm/versions/node/v16.14.0/bin/node
```

Comment:
perhaps `~/.nvm/versions/node/v16.14.0/bin/node` to have more chance of running 'out of the box' ?

Also, what if node is already in the path?  Maybe it would be better if it was empty by default..

Reply:
updated!